### PR TITLE
Codacy error cleanup

### PR DESCRIPTION
**Pull request by red15**
Some cleanup from the codacy *Error Prone* category.

### CODE REVIEW

File: app/common/gmapService/gmapService.js [hunk 1]
```diff
@@ -253,7 +252,7 @@
 
 			// Generate final SIDC string
 			var _sidc = "";
-			for (atr in _sidcObject) {
+			for (var atr in _sidcObject) {
```
[the remark this answers is not in the document]
Comment (red15) [reply app/common/gmapService/gmapService.js]: Fixed in cd08cf9 commit

File: app/common/gmapService/gmapService.js [hunk 2]
```diff
@@ -253,8 +252,8 @@
 
 			// Generate final SIDC string
 			var _sidc = "";
-			for (atr in _sidcObject) {
-				_sidc += _sidcObject[atr];
+			for (var sideatr in _sidcObject) {
+				_sidc += _sidcObject[sideatr];
```
[the remark this answers is not in the document]
Comment (red15) [reply app/common/gmapService/gmapService.js]: @afinegan 
anything you can tell me about your SIDC string ? is the _sidc value meant for showing as text on the map or something? How do you guarantee the order of the attributes appearing in the right order in the string ?

[the remark this answers is not in the document]
Comment (red15) [reply app/common/gmapService/gmapService.js]: Looking to replace this code with
```suggestion
// Generate final SIDC string                   
var _sidc = Object.values(_sidcObject).join('');
```

[the remark this answers is not in the document]
Comment (afinegan) [reply app/common/gmapService/gmapService.js]: sidc is how the units choose the proper military symbol when displayed on the google map, all this stuff is turned off due to cheaters,https://github.com/spatialillusions/milsymbol - there is a metric ton of code in this project that can be cleaned, I see stuff every time I work on it

[the remark this answers is not in the document]
Comment (red15) [reply app/common/gmapService/gmapService.js]: I meant mostly I'm confused by how you combined all the flags and if the order of these flags matter or not ?

**Pull request closed without merging**